phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardFactory.java

```diff
@@ -23,6 +23,7 @@
 import org.springframework.beans.factory.annotation.Value;
 
 import java.net.URI;
+import java.util.Optional;
 
 public class UiSetupWizardFactory {
```

Comment:
JavaDoc

Reply:
Don't know a useful comment which I could add to the class but commented the methods.